cypress/support/step-definitions/flat-table.js

```diff
@@ -0,0 +1,163 @@
+import {
+  flatTableRowHeader, flatTableHeader, flatTableHeadRow,
+  flatTableBodyRow, flatTableHeaderCell, flatTabelCell, flatTable,
+} from '../../locators/flat-table';
+
+const FIRST_ELEMENT = 0;
+const SECOND_ELEMENT = 1;
+const THIRD_ELEMENT = 2;
+const FOURTH_ELEMENT = 3;
+const FIFTH_ELEMENT = 4;
+const SIXTH_ELEMENT = 5;
+const SEVENTH_ELEMENT = 6;
+const EIGHT_ELEMENT = 7;
+const NIGHT_ELEMENT = 8;
+
+Then('FlatTable has row header', () => {
```

Comment:
could we use `const color = \'rgb(204, 214, 218)\'` and use it in all assertions here in test?